src/components/PriceLine.tsx

```diff
@@ -73,7 +74,7 @@ const PriceLine = ({
                 )}
                 <div className='draggable-area' />
                 <div className='drag-price' style={{ backgroundColor: color, width, opacity }}>
-                    <div className='price'>{priceDisplay}</div>
+                    <div className='price'>{isDragging? priceDisplay : price}</div>
```

Comment:
this condition allows to show correct number while dragging the barriers in high/low and touch/ no touch contracts.
because while dragging price became not rounded and looks ugly (f.e: 9899,4634762347624 instead of 9899,463 having a pipsize 3)

Reply:
Hey @yauheni-deriv , is this change requested in your card? I think it make sense to have a more specific value while dragging the barriers to be able to drag it more accurately. Unless the number of decimals is indeed too big.

Reply:
without these changes we couldn’t see real BE values. Because 'priceDisplay' consider pipsize of the symbol. and for other contracts which require barriers to be adjustable the behaviour will remain the same while dragging.
And if there are some issues with BE values for barriers of other contracts BE will fix it as was agreed